src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +22,57 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      searchText: '',
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSearch(event) {
+    this.setState({
+      searchText: event.target.value,
+    });
+  }
+
+  handleSelect({ option }) {
+    this.setState({
+      translationLanguage: option,
+    });
+  }
+
+  getSelectOptions() {
+    const searchStr = this.state.searchText;
+    let langs = languages.slice();
+    if (searchStr) {
+      const matchesSearch = (langObj) => langObj.name.toLowerCase().substr(0, searchStr.length) === searchStr;
+      langs = languages.filter(matchesSearch);
+    }
+    return langs.map(item => item.name);
```

Comment:
When I select 'English' from the menu, the select's value is 'English', but it should be 'en'. I think each option needs to have a label and a value set.